### DISCUSSION TITLE
Bahama Coleman question

### DISCUSSION THREAD

**Discussion by lyonrt** [visualapproach/WiFi-remote-for-Bestway-Lay-Z-SPA#309] [Q&A]
I have the Coleman spa from last year(square shape pump)  Bahama model.  Does anyone have pictures how to open up that particular square pump and what #of wires?  Don't want to make the wife mad and want to be sure I do it right the first time :-).

**Comment by visualapproach** [comment]
There are some pictures and information about this somewhere on this forum, that other people has kindly posted.
